katello.dot

```diff
@@ -2,108 +2,120 @@
 
 [Browser] as browser
 
-node "server" {
-  [Postgres] as postgres
-  [Candlepin] as candlepin
+node "Main Environment" {
+	
+	node "Server" {
+	  [Postgres] as postgres
+	  
 
-  [Elasticsearch] as elasticsearch
-  [MongoDB] as mongo_server
-  [Qpid] as qpidd_server
-  [Dispatch Router] as qdrouterd_server
-  [Smart Proxy] as server_smart_proxy
+	  [MongoDB] as mongo_server
+	  [Qpid] as qpidd_server
+	  [Dispatch Router] as qdrouterd_server
+	  [Smart Proxy] as server_smart_proxy
+	  
+	  node "Tomcat" as tomcat_server {
+		  [Candlepin] as candlepin
+	  }
 
-  node "Apache" as server_apache {
+	  node "Apache" as server_apache {
 
-    node "Pulp" as pulp_server {
-      [API] as pulp_server_api
-      [pulp_* processes] as pulp_server_backend
-    }
+		node "Pulp" as pulp_server {
+		  [API] as pulp_server_api
+		  [pulp_* processes] as pulp_server_backend
+		}
 
-    [Puppet Master] as puppet_master
+		[Puppet Master] as puppet_master
```

Comment:
I suppose we should drop this in favor of Puppet Server (for Puppet 4 and 5) and move it outside of Apache since it's stand alone Java now.